src/tools/ceph_dedup_tool.cc

```diff
@@ -1574,6 +1615,11 @@ int make_crawling_daemon(const po::variables_map &opts)
     cout << "100 second is set as wakeup period by default" << std::endl;
   }
 
+  unsigned fp_threshold = DEFAULT_MEMORY_THRESHOLD;
+  if (opts.count("fpstore-threshold")) {
+    fp_threshold = opts["fpstore-threshold"].as<int>();
```

Comment:
Isn't there a way to define a default for an option where the option is defined?